app/components/loading-indicator/component.js

```diff
@@ -0,0 +1,4 @@
+import Component from '@ember/component';
+
+export default Component.extend({
```

Comment:
You can probably jut put `classNames: ['ball-scale', 'ball-light', 'LoadingIndicator']` here and avoid the template altogether

Reply:
You still need the empty internal div. Also, note [RFC 278](https://github.com/emberjs/rfcs/blob/master/text/0278-template-only-components.md), which is only for template-only components, but represents the direction Ember is headed.